tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/provider/TieredStorageConfiguration.java

```diff
@@ -329,6 +329,13 @@ protected Properties getOverrides() {
             overrides.setProperty(S3Constants.PROPERTY_S3_VIRTUAL_HOST_BUCKETS, "false");
         }
 
+        // load more jclouds properties into the overrides
+        System.getProperties().entrySet().stream()
```

Comment:
`System.getenv()` instead of `System.getProperties()` ?

Reply:
Add both of them.